modules/gcp-vpn.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+//Specify the module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/gcp-private-connections.adoc
+
+[id="gcp-vpn_{context}"]
+= Configuring a GCP Cloud VPN
+
+[role="_abstract"]
+You can configure a {GCP} {product-title} cluster to securely connect your High Availability (HA) Virtual Private Network (VPN) to your Virtual Private Cloud (VPC) network through an IPsec tunnel.
```

Comment:
I would suggest to change it to:

You can configure a High Availability (HA) Virtual Private Network (VPN) to securely connect your peer network to your {GCP} {product-title} cluster through an IPsec tunnel.